packages/scan/src/requests/UploadFile.ts

```diff
@@ -0,0 +1,23 @@
+import FormData from 'form-data';
+import { HttpOptions, HttpRequest } from '@secbox/bus';
+
+export interface UploadFileOptions {
+  filename: string;
+  content: string;
+}
+
+export class UploadFile extends HttpRequest<FormData, { id: string }> {
```

Comment:
Are you going to use this class to upload files other than HAR? Nope 😄 
```suggestion
export class UploadHar extends HttpRequest<FormData, { id: string }> {
```